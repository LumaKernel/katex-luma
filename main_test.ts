import { assertSnapshot } from "@std/testing/snapshot";
import { katexLumaRenderToString } from "./main.ts";

Deno.test({
  name: "katexLumaRenderToStringTest",
  async fn(t) {
    await assertSnapshot(t, katexLumaRenderToString(String.raw`\tau_1`));
  },
  permissions: {
    read: true,
    write: ["__snapshots__"],
  },
});
