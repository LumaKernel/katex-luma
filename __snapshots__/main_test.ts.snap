export const snapshot = {};

snapshot[`katexLumaRenderToStringTest 1`] = `'<span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>τ</mi><mn>1</mn></msub></mrow></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5806em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathnormal" style="margin-right:0.1132em;">τ</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3011em;"><span style="top:-2.55em;margin-left:-0.1132em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">1</span></span></span></span><span class="vlist-s">​</span></span></span></span></span></span></span></span>'`;
